app/models/conversion_host.rb

```diff
@@ -274,16 +277,39 @@ def miq_ssh_util_args_manageiq_providers_openstack_cloudmanager_vm
   # +extra_vars+ option should be a hash of key/value pairs which, if present,
   # will be passed to the '-e' flag.
   #
-  def ansible_playbook(playbook, extra_vars = {})
-    command = "ansible-playbook #{playbook} -i #{ipaddress}"
+  def ansible_playbook(playbook, extra_vars = {}, auth_type = nil)
+    host = hostname || ipaddress
+
+    command = "ansible-playbook #{playbook} --inventory #{host}, --become -vvv"
+
+    auth = authentication_type(auth_type) || authentications.first
+    command += " --user #{auth.userid}"
+
+    case auth
+    when AuthUseridPassword
+      extra_vars[:ansible_ssh_pass] = auth.password
+    when AuthPrivateKey
+      ssh_private_key_file = Tempfile.new('ansible_key')
+      ssh_private_key_file.write(auth.auth_key)
+      command += " --private-key #{ssh_private_key_file.path}"
+    else
+      raise MiqException::MiqInvalidCredentialsError, _("Unknown auth type: #{auth.authtype}")
+    end
+
 
-    extra_vars[:v2v_host_type] = resource.ext_management_system.emstype
-    extra_vars.each { |k, v| command += " -e '#{k}=#{v}'" }
+    extra_vars.each { |k, v| command += " --extra-vars '#{k}=#{v}'" }
 
-    connect_ssh { |ssu| ssu.shell_exec(command) }
+    _log.info("FDUPONT - Calling Ansible playbook: #{command}")
```

Comment:
Don't forget to remove these FDUPONT debug lines

Reply:
Yes, and also because they might contain some sensitive data.